src/Umbraco.Core/Notifications/UmbracoApplicationStoppedNotification.cs

```diff
@@ -5,5 +5,26 @@ namespace Umbraco.Cms.Core.Notifications
     /// </summary>
     /// <seealso cref="Umbraco.Cms.Core.Notifications.INotification" />
     public class UmbracoApplicationStoppedNotification : INotification
-    { }
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UmbracoApplicationStoppedNotification" /> class.
+        /// </summary>
+        public UmbracoApplicationStoppedNotification()
+            : this(false)
+        { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UmbracoApplicationStoppedNotification" /> class.
+        /// </summary>
+        /// <param name="isRestarting">Indicates whether Umbraco is restarting.</param>
+        public UmbracoApplicationStoppedNotification(bool isRestarting) => IsRestarting = isRestarting;
```

Comment:
Same as the `UmbracoApplicationStartedNotification`:
```suggestion
        /// <summary>
        /// Initializes a new instance of the <see cref="UmbracoApplicationStoppedNotification" /> class.
        /// </summary>
        /// <param name="isRestarting">Indicates whether Umbraco is restarting.</param>
        public UmbracoApplicationStoppedNotification(bool isRestarting = false) => IsRestarting = isRestarting;
```